packages/babel-traverse/src/scope/lib/renamer.js

```diff
@@ -9,8 +9,9 @@ const renameVisitor = {
     }
   },
 
-  Scope(path, state) {
+  "Pattern|Scope"(path, state) {
```

Comment:
Since now the correct `Pattern`s are already scope, aren't they already matched by the `Scope` visitor?